resources/velero/templates/NOTES.txt

```diff
@@ -0,0 +1,4 @@
+Once the Velero server is up and running, you need the client before you can use it.
```

Comment:
Shouldn't that be in the readme? Or is it linked somewhere?

Reply:
IMO it is helpful to be in the NOTES.txt as it appears right after installation of the chart. At the same time I added an extra point on Velero CLI in the chart README.md.